features/support/hooks.rb

```diff
@@ -72,6 +72,10 @@
   end
 end
 
+Before('@javascript') do
+  Rails.application.config.active_job.enqueue_after_transaction_commit = true
+end
+
```

Comment:
Is this because this isn't getting set correctly during the Cucumber tests?

It might be worth leaving a comment here to say that, after the defaults have been changed to 7.2 in `config/application.rb` it might be possible to remove this.